aio_pika/channel.py

```diff
@@ -65,6 +66,14 @@ def __init__(self, connection, channel_number: int = None,
         # noinspection PyTypeChecker
         self.default_exchange = None       # type: Exchange
 
+    @property
+    def done_callbacks(self) -> CallbackCollection:
```

Comment:
Why do you wrap callback collections in properties?

Reply:
for restrict setter behaviour